authbridge/demos/weather-agent/demo-ui-advanced.md

```
@@ -49,34 +48,34 @@ cd authbridge
 python demos/weather-agent/setup_keycloak_weather_advanced.py \
   -n team1 --wait-tool-client
 ```
 
-`--wait-tool-client` blocks until the tool pod (Step 2) registers its SPIFFE
-client. **Re-run the same command after Step 3** so the agent client picks up
-the optional exchange scope. The script:
+> **Do not `kubectl port-forward` Keycloak on `:8080`.** `keycloak.localtest.me:8080`
+> is already reachable directly; a forward on local `:8080` hijacks the Rossoctl UI
+> (`rossoctl-ui.localtest.me:8080`) into the Keycloak admin console. If you must
+> forward, use another local port and set `KEYCLOAK_URL` to match.
+
```

markdownlint-cli2 (0.23.2)

[warning] Blank line inside blockquote

(MD028, no-blanks-blockquote)